storage/src/main/java/org/apache/kafka/storage/internals/log/RemoteIndexCache.java

```diff
@@ -716,4 +710,8 @@ public static String remoteTransactionIndexFileName(RemoteLogSegmentMetadata rem
         return generateFileNamePrefixForIndex(remoteLogSegmentMetadata) + LogFileUtils.TXN_INDEX_FILE_SUFFIX;
     }
 
+    public static String remoteDeletedSuffixIndexFileName(RemoteLogSegmentMetadata remoteLogSegmentMetadata) {
```

Comment:
please add a comment:
`// visible for testing`

Reply:
updated